Dockerfile

```diff
@@ -12,13 +12,57 @@ LABEL changelog-url "https://github.com/CommunityHoneyNetwork/communityhoneynetw
 VOLUME /tls
 
 ENV playbook "chnserver.yml"
+ENV TZ "America/New_York"
+ENV DEBIAN_FRONTEND "noninteractive"
 
 RUN date
-RUN apt-get update \
-      && apt-get install -y ansible
+RUN apt-get update
+
+RUN apt-get install -y gcc git nginx python3-pip python3-dev redis-server libgeoip-dev libsqlite3-dev runit python3-certbot-nginx net-tools jq curl
+
+ADD requirements.txt /opt/requirements.txt
+RUN pip3 install -r /opt/requirements.txt
+
+# Make Nginx directories
+RUN mkdir -p /opt/www /etc/nginx /etc/nginx/sites-available /etc/nginx/sites-enabled /etc/pki/tls/certs /etc/pki/tls/private
+
+# sites-enable link
+RUN ln -sf /etc/nginx/sites-available/default /etc/nginx/sites-enabled/default
+
+# Create vhost conf
+COPY nginx_vhost.conf /etc/nginx/sites-available/default
+
+# Create nginx server conf
+COPY nginx.conf /etc/nginx/nginx.conf
+
+# Create runit directories
+RUN mkdir -p /etc/service/nginx /etc/service/uwsgi
+
+# Create nginx runit conf
+COPY nginx.run /etc/service/nginx/run
+RUN chmod 755 /etc/service/nginx/run
+
+# Create CHNServer sysconfig file
+COPY chnserver.sysconfig /etc/default/chnserver
+
+# Create uwsgi runit conf
+COPY uwsgi.run /etc/service/uwsgi/run
+RUN chmod 755 /etc/service/uwsgi/run
+
+# Create log file for uwsgi
+RUN mkdir -p /var/log/mhn \
+    && chown www-data /var/log/mhn \
+    && chmod 0755 /var/log/mhn
+
+# Create sqlite directory
+RUN mkdir -p /opt/sqlite
+
+# Link to hpfeeds in chnserver
+RUN pip3 install git+https://github.com/CommunityHoneyNetwork/hpfeeds3.git
+
+# Link chnctl.py to bindir
+RUN ln -s /opt/chnctl.py /usr/local/bin/chnctl
 
-RUN echo "localhost ansible_connection=local" >> /etc/ansible/hosts
 ADD . /opt/
-RUN ansible-playbook /opt/${playbook}
 
 ENTRYPOINT ["/usr/bin/runsvdir", "-P", "/etc/service"]
```

Comment:
No issue, but a note for both of use that we should explore moving the nginx portion of this into it's own container? this type of init process won't work for k8s.